Tests/Marketplace/search_and_install_packs.py

```diff
@@ -690,18 +690,27 @@ def search_and_install_packs_and_their_dependencies(pack_ids: list,
 
     lock = Lock()
 
-    with ThreadPoolExecutor(max_workers=130) as pool:
+    if not install_packs_one_by_one:
```

Comment:
Why not change it to be more straight forward, and positive? :-)
`if not install_packs_one_by_one` -> `if install_packs_one_by_one`
And swap the `if` and `else` blocks?

Reply:
Positive is me! I will change it thanks.